src/styles/blocks/main.scss

```diff
@@ -0,0 +1,22 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  grid-auto-flow: row;
+  gap: 46px 48px;
+
+  justify-content: center;
+
+  @include on-screen2 {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @include on-screen3 {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @include on-screen4 {
```

Comment:
Ensure that the mixin `on-screen4` is defined in your stylesheets. If not, you need to define it to apply the intended styles for different screen sizes.